en/config_mc/pid_tuning_guide_multicopter_basic.md

```diff
@@ -0,0 +1,93 @@
+Make sure to have assigned a [Kill switch](../config/safety.md#emergency-switches).
+:::
+
+## Introduction
+
+PX4 uses **P**roportional, **I**ntegral, **D**erivative (PID) controllers (these are the most widespread control technique).
+
+The _QGroundControl_ **PID Tuning** setup provides real-time plots of the vehicle setpoint and response curves. 
+The goal of tuning is to set the P/I/D values such that the _Response_ curve matches the _Setpoint_ curve as closely as possible (i.e. a fast response without overshoots).
+
+![QGC Rate Controller Tuning UI](../../assets/mc_pid_tuning/qgc_mc_pid_tuning_rate_controller.png)
+
+The controllers are layered, which means a higher-level controller passes its results to a lower-level controller.
+The lowest-level controller is the **rate controller**, followed by the **attitude controller**, and finally the **velocity & position controller**.
+The PID tuning needs to be done in this same order, starting with the rate controller, as it will affect all other controllers.
+
+The testing procedure for each controller (rate, attitude, velocity/posibition) and axis (yaw, roll, pitch) is always the same: create a fast setpoint change by moving the sticks very rapidly and observe the response.
+Then adjust the sliders (as discussed below) to improve the tracking of the response to the setpoint.
+
+:::tip
+- Rate controller tuning is the most important, and if tuned well, the other controllers often need no or only minor adjustments
+- Usually the same tuning gains can be used for roll and pitch.
+- use Acro/Stabilized/Altitude mode to tune the rate controller
+- Use Position mode to tune the velocity and position controllers.
+  Make sure to switch to the simple input mode, for direct control and possibility to generate step inputs.
+:::
+
+## Preconditions
+
+- You have selected the closest matching [default airframe configuration](../config/airframe.md) for your vehicle.
+  This should give you a vehicle that already flies.
+- You should have done an [ESC calibration](../advanced_config/esc_calibration.md).
+- If using PWM output: [PWM_MIN](../advanced_config/parameter_reference.md#PWM_MIN) is set correctly.
+  It needs to be set low, but such that the **motors never stop** when the vehicle is armed.
+
+  This can be tested in [Acro mode](../flight_modes/acro_mc.md) or in [Manual/Stabilized mode](../flight_modes/manual_stabilized_mc.md):
+  - Remove propellers
+  - Arm the vehicle and lower the throttle to the minimum
+  - Tilt the vehicle to all directions, about 60 degrees
+  - Check that no motors turn off
+- Use a high-rate telemetry link such as WiFi if at all possible (a typical low-range telemetry radio is not fast enough for real-time feedback and plots).
+  This is particularly important for the rate controller.
+- Disable [MC_AIRMODE](../advanced_config/parameter_reference.md#MC_AIRMODE) before tuning a vehicle.
+
+## Tuning Procedure
+
```

Comment:
This is now a true "step by step" procedure. I like doing this even though it is less compact because it makes you think about what order people should do things and what is documented vs not documented. 

Some questions from this
-ie what is thrust curve? What value should be set?
- When are the values saved? 
- What are clipboard values used for?
- When should I clear or start ?
- What does automatic flight mode switching do. When should I use it?

Reply:
> -ie what is thrust curve? What value should be set?

I'd point to the advanced page, as it requires using logs if you want to tune it really accurate. But w/o any detailed tuning, 0.3 should generally be a good value (the difficulty is that we can't just change the default, as it would invalidate existing setups)

> When are the values saved?

Immediately

> What are clipboard values used for?

To restore the previous gains

> When should I clear or start ?

Well that's really up to you. When you start zooming/panning, it stops, so you want to start before doing the next iteration (it auto starts on arming).

> What does automatic flight mode switching do. When should I use it?

This is if you have very little flying experience. You can keep it in position control, and when you press start, it will switch to stabilized. This is something we'll have to see how useful it is in practise.